docs/dev-setup.md

```diff
@@ -233,18 +233,18 @@ The output can be passed directly to `kubectl apply` like this:
 go run cmd/make-virt-host/main.go openshift_worker_1 | kubectl apply -f -
 ```
 
-When the host is a *master*, include the `-consumer` and
+When the host is a *controlplane*, include the `-consumer` and
```

Comment:
nit: I would say "a controlplane host" or "part of the control plane".

Reply:
Done